api/server/restapi/configure_massa_wallet.go

```diff
@@ -82,11 +82,18 @@ func configureAPI(api *operations.MassaWalletAPI) http.Handler {
 			return middleware.NotImplemented("operation operations.RestAccountSignOperation has not yet been implemented")
 		})
 	}
+	// route for the legacy wallet frontend
 	if api.WebHandler == nil {
 		api.WebHandler = operations.WebHandlerFunc(func(params operations.WebParams) middleware.Responder {
 			return middleware.NotImplemented("operation operations.Web has not yet been implemented")
 		})
 	}
+	// route for the ReactJS wallet frontend
```

Comment:
i don't really understand why you add this... its actually implemented

Reply:
I will delete it in another PR, thanks